### ISSUE TITLE
Support run on multiple solutions

### ISSUE THREAD

**Issue by imanushin** [Litee/SolutionCop#32]
Sometimes whole project has different solutions (to speedup development). In this case, solution cop should be run on the all of them (of course, rules are the same, however each solution is checked independently).

Nice to have: parallel check, instead of sequental check.

**Comment by Litee** [comment]
How much time will you win from parallel checks in your case?

**Comment by imanushin** [comment]
@Litee , I'd like to have one script for all solutions. For my opinion it is better to run all in parallel which is a minor improvement for virtualized environment (when VM host uses HDD, which is virtualized and shared between separate VMs, moreover sometimes different agents works on the same computer, and IO is a bottle neck)
